kedro-datasets/pyproject.toml

```diff
@@ -215,8 +215,9 @@ test = [
     "deltalake>=0.10.0",
     "dill~=0.3.1",
     "filelock>=3.4.0, <4.0",
+    "fiona >=1.8, <2.0",
     "gcsfs>=2023.1, <2023.3",
-    "geopandas>=0.6.0, <1.0",
+    "geopandas>=0.8.0, <2.0",
```

Comment:
I'll let others who were already looking into this review, but thank your for this! ❤️